src/sirinternal.c

```diff
@@ -471,8 +483,13 @@ bool _sir_mapmutexid(sir_mutex_id mid, sir_mutex** m, void** section) {
             tmpsec = &_sir_pc;
             break;
         case SIRMI_TEXTSTYLE:
+#if !defined(SIR_NO_TEXT_STYLING)
             tmpm   = &ts_mutex;
             tmpsec = &sir_text_style_section;
+#else
```

Comment:
I don't know what the side effects of this will be unless we have also removed all `_sir_locksection` (`_SIR_LOCK_SECTION`) for `SIRMI_TEXSTYLE`, so we better check that

Reply:
I checked it ... and verified that's the case now.